src/components/ApplicationGroup/AppGroup.service.ts

```diff
@@ -189,27 +191,17 @@ export const getConfigAppList = (envId: number, appIds: string): Promise<ConfigA
     return get(`${Routes.ENVIRONMENT}/${envId}/${Routes.ENV_APPLICATIONS}${getFilteredAppQueryString(appIds)}`)
 }
 
-export const getEnvAppList = (
-    params?: {
-        envName?: string
-        clusterIds?: string
-        offset?: string
-        size?: string
-    },
-    signal?: AbortSignal,
-): Promise<EnvAppType> => {
+export const getEnvAppList = (filterConfig?: Partial<AppGroupFilterConfig>, signal?: AbortSignal): Promise<EnvAppType> => {
     const options = signal ? { signal } : null
+    const {searchKey, cluster, offset, pageSize} = filterConfig
```

Comment:
If filterConfig is optional in params, we may need to add fallback before de-structuring

Reply:
Not yet resolved, we also need to filterConfig || {}